### PR TITLE
restructure directories - UI

### PR DESCRIPTION
**Pull request by DanielOsypenko**
Due to proposed directory structure and email "Test Directory Restructure Next Steps" from 5 Oct 2023

### CODE REVIEW

**Review by am-agrawa — APPROVED** [review]

File: tests/cross_functional/ui/test_pvc_ui.py
Comment (am-agrawa) [inline tests/cross_functional/ui/test_pvc_ui.py:1]: should be functional IMO

File: tests/functional/ui/test_capacity_breakdown_ui.py
Comment (am-agrawa) [inline tests/functional/ui/test_capacity_breakdown_ui.py:1]: should be functional IMO

**Review by am-agrawa — COMMENTED** [review]
test_pvc_ui.py and test_pv_encryption_ui.py should be functional IMO

**Review by keesturam — APPROVED** [review]

**Comment by DanielOsypenko** [comment]
closing this, created similar PR 
https://github.com/red-hat-storage/ocs-ci/pull/9151

**Pull request closed without merging**